package.json

```diff
@@ -14,6 +14,7 @@
     "@astrojs/starlight-tailwind": "^2.0.1",
     "@astrojs/tailwind": "^5.1.0",
     "astro": "^4.0.8",
+    "pnpm": "^8.15.4",
```

Comment:
Why is this here?

Reply:
Not sure, I'll just copy paste the current package.json file into this branch's

Reply:
There shouldn't be a package-lock.json and the pnpm lock should not have been updated in this PR so can you revert those please so that those 3 files have no changes in this PR

Reply:
I think I got it this time